examples/asr/notebooks/6_VAD_using_NeMo.ipynb

```diff
@@ -0,0 +1,991 @@
+    " \n",
+    " - Model training on the Google Speech Commands dataset and Freesound dataset in NeMo.\n",
+    " \n",
+    " - Evaluation of error cases of the model by audibly hearing the samples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Preparation\n",
+    "\n",
+    "## Download the Freesound dataset\n",
+    "We will be using the freesound dataset (background categories) as our non-speech/background data. \n",
+    "We provide scripts and you can customize a lot by using it. Note that downloading this dataset may takes hours. \n",
+    "\n",
+    "1. Download provided downloading and resampling scripts"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/freesound_download_resample/download_resample_freesound.sh\n",
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/freesound_download_resample/freesound_download.py\n",
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/freesound_download_resample/freesound_private_apikey.py\n",
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/freesound_download_resample/freesound_resample.py\n",
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/freesound_download_resample/freesound_requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. We will need some requirements including freesound, requests, requests_oauthlib, joblib, librosa and sox. If they are not installed, please run "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -r freesound_requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "3. Create an API key for freesound.org at https://freesound.org/help/developers/ and paste the cliend_id and api_key to freesound_private_apikey\n",
+    "4. Authorize by run python freesound_download.py --authorize and visit website and paste response code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python freesound_download.py --authorize"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "5. Feel free to change any arguments for freesound_download.py in **download_resample_freesound.sh** such as max_samples and max_filesize\n",
+    "6. Run `bash download_resample_freesound.sh <max number of samples you want> <download data directory> <resampled data directory> `"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!bash download_resample_freesound.sh 4000 ./freesound  ./freesound_resampled_background"
```

Comment:
Dont use 4000 samples in the notebook, it will take far too long to download and notebooks on colab time out after 8 hours or so. Set the number to a manageable 100 or so. The notebook is just a demonstration of the approach, not full fledged training script.